graphql/models/UserProfile.graphql

```diff
@@ -31,8 +31,10 @@ type CompanyBranch {
     id: ID!
     name: String!
     location: String
+    company_balance: Float!
```

Comment:
@Oluwablin 
This field should be renamed to "branch_balance", it's more specific that way.

Reply:
Okay, sir.

Reply:
> @Oluwablin
> This field should be renamed to "branch_balance", it's more specific that way.

@omadoyeabraham done